deploy/platform/kubernetes/Makefile

```diff
@@ -147,6 +147,14 @@ feature-elasticsearch-monitor:
 	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set prometheus-elasticsearch-exporter.es.uri=http://elasticsearch-master:9200)
 	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set opentelemetry.enabled=true)
 
+.PHONY: feature-rabbitmq-monitor
+feature-rabbitmq-monitor:
+	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set features.rabbitmqMonitor.enabled=true)
+	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set rabbitmq.replicaCount=1)
+	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set rabbitmq.persistence.enabled=false)
+	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set rabbitmq.metrics.enabled=true)
```

Comment:
Let's put these into the `values.yaml` file and keep the Makefile clean

Reply:
Can you also move these to `values.yaml` file too? I missed these before:

```
	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set prometheus-elasticsearch-exporter.fullnameOverride=prometheus-elasticsearch-exporter)
	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set prometheus-elasticsearch-exporter.service.httpPort=9114)
	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set prometheus-elasticsearch-exporter.es.uri=http://elasticsearch-master:9200)
	$(eval HELM_OPTIONS := $(HELM_OPTIONS) --set opentelemetry.enabled=true)
```

Reply:
done.